src/plugins/kibana_utils/public/history/ensure_default_index_pattern.tsx

```diff
@@ -41,8 +40,7 @@ let timeoutId: NodeJS.Timeout | undefined;
 export async function ensureDefaultIndexPattern(
   newPlatform: CoreStart,
```

Comment:
Nit, could you rename `newPlatform`to `core` since we're already in NP?

Reply:
yeah, makes sense !
Just was hesitant to rename it after Joe gave this name 😃